.gitignore

```diff
@@ -24,9 +22,8 @@ perf.data.old
 tags
 .lock-wscript
 *.pyc
-doc/api.xml
+/doc/api.xml
```

Comment:
why is this not an absolute path?

Reply:
Do you mean why _was_ this not an absolute path? This PR makes it one right?

Reply:
I made all relative paths absolute, since we have all sorts of path patterns deeper in the tree.